woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -2456,7 +2459,11 @@ def test_accessor_schema_properties(sample_df):
         assert prop_from_accessor == prop_from_schema
 
         # Assumes we don't have setters for any of these attributes
-        error = "can't set attribute"
+        if Version(python_version().strip(punctuation)) < Version("3.11.0"):
+            error = "can't set attribute"
+        else:
+            # Error messages were changed as part of Python 3.11
+            error = "has no setter"
         with pytest.raises(AttributeError, match=error):
```

Comment:
is there a reason for the match parameter here? If it throws the same built-in python exception, it doesn't really matter what the message is. Id rather delete this param and all the custom logic above it. No reason to look for it imo

Reply:
Yeah, I agree with @ozzieD here. If we were checking for a custom error we were raising, I think we would want to use `match`, but since we are just checking that a built-in exception gets raised, I'm not sure there is much value in adding complexity to the test and checking the actual error message since that is beyond our control.

Reply:
I'm not too sure, maybe to disambiguate between different types of AttributeError?

Reply:
I think just checking for the attribute error should be fine. We already know that the attribute exists on the schema and accessor objects, so we shouldn't have to worry that an attribute error would pop up because the attribute doesn't exist, which might have been what I was trying to differentiate from here? Probably just an oversight on my part--I'm totally fine with removing the matching.